### PR TITLE
Added Lorem Ipsum Generator tool

### PR DESCRIPTION
**Pull request by ivankahl**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Internationalization and localization
- [ ] Other (please describe):

## What is the current behavior?

There is currently no Lorem Ipsum tool.

Issue Number: #73 

## What is the new behavior?

- Created a new tool under the Generators section for Lorem Ipsum
- Tool can generate Lorem Ipsum text with the following parameters
   - **Type:** Words, Sentences, Paragraphs
   - **Length:** How many words/sentences/paragraphs to generate

### NLipsum

I have used the NLipsum library to generate the Lorem Ipsum text. It is quite feature-rich so we can extend this tool later if need be and still use NLipsum.

**NLipsum is also licensed under the MIT license.**

#### Icon in All Tools screen

Icon in All Tools screen

#### Tool in navigation menu

Tool in navigation menu

#### Demo video

https://user-images.githubusercontent.com/5931577/147853153-43050534-314b-4c96-808e-a701587540e1.mp4

## Quality check

Before creating this PR, have you:

- [x] Followed the code style guideline as described in [CONTRIBUTING.md](https://github.com/veler/DevToys/blob/main/CONTRIBUTING.md)
- [x] Verified that the change work in Release build configuration
- [x] Checked all unit tests pass

### CODE REVIEW

**Review by veler — CHANGES_REQUESTED** [review]
Thank you so much for this! Overall, it's excellent :D I will let @btiteux taking a look at it too.

File: src/dev/impl/DevToys/DevToys.csproj [hunk 1]
```diff
@@ -562,6 +574,9 @@
     <PackageReference Include="Microsoft.Toolkit.Uwp.UI.Controls" Version="$(Nuget_MicrosoftToolkitUwpUIControls)" />
     <PackageReference Include="Microsoft.UI.Xaml" Version="$(NuGet_MicrosoftUIXaml)" />
     <PackageReference Include="Newtonsoft.Json" Version="$(NuGet_NewtonsoftJson)" />
+    <PackageReference Include="NLipsum">
```
[the remark this answers is not in the document]
Comment (ivankahl) [reply src/dev/impl/DevToys/DevToys.csproj]: Thanks for the feedback @veler. Neat trick with the NuGet versions! I've made the changes 😃👍

File: src/dev/impl/DevToys/DevToys.csproj [hunk 2]
```diff
@@ -562,6 +574,9 @@
     <PackageReference Include="Microsoft.Toolkit.Uwp.UI.Controls" Version="$(Nuget_MicrosoftToolkitUwpUIControls)" />
     <PackageReference Include="Microsoft.UI.Xaml" Version="$(NuGet_MicrosoftUIXaml)" />
     <PackageReference Include="Newtonsoft.Json" Version="$(NuGet_NewtonsoftJson)" />
+    <PackageReference Include="NLipsum" Version="$(NuGet_NLipsum)">
+      <Version>1.1.0</Version>
```
Comment (veler) [inline src/dev/impl/DevToys/DevToys.csproj:578]: You should be able to remove this line, since you set the version number on the line above :)

**Review by veler — CHANGES_REQUESTED** [review]

File: src/dev/impl/DevToys/DevToys.csproj (hunk 1 above)
[the remark this answers is not in the document]
Comment (veler) [reply src/dev/impl/DevToys/DevToys.csproj]: Awesome, thank you! Just an additional nit comment and we're good to go :D

File: src/dev/impl/DevToys/DevToys.csproj (hunk 2 above)
Comment (ivankahl) [reply src/dev/impl/DevToys/DevToys.csproj]: Ah good catch! Whoops! Fixed it now :)

**Review by veler — APPROVED** [review]
Awesome! Thank you so much! :D Merging it in main branch, but we will release it in an update after 1.0.0.0 :)

**Pull request merged**